### DISCUSSION TITLE
Whitelist

### DISCUSSION THREAD

**Discussion by tizu69** [uBlockOrigin/uBOL-home#239] [Q&A] [answered]
currently only site blacklist is possible, why? :3

**Comment by gorhill** [comment]
I don't understand what you are asking.

**Reply by tizu69** [reply]
sorry for my bad english. basically blacklist, but you provide a domains you want to adblock, and the rest gets ignored and shows ads., as a mode

**Reply by gorhill** [reply, marked as answer]
Add `all-urls` to _No filtering_ field. See <https://github.com/uBlockOrigin/uBOL-home/issues/150#issuecomment-2233379095>.
